bodhi-messages/bodhi-messages.spec

```diff
@@ -2,7 +2,7 @@
 %global pypi_name bodhi-messages
 %global pypi_version 5.7.4
 
-Name:           %{pypi_name}
+Name:           python3-%{pypi_name}
```

Comment:
This doesn't conform with the [Fedora Python Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_library_naming): for a package containing primarily a Python library, the source package (aka the "component") must be named `python-…`, and the built package must be named `python3-…`. This means we need a new `%package python3-%{pypi_name}` section.

Reply:
I added this section -- i think it is correct :)